Code/Framework/AzCore/Platform/Windows/AzCore/IO/Streamer/StorageDrive_Windows.cpp

```diff
@@ -1153,21 +1189,78 @@ namespace AZ::IO
     {
         switch (data.m_reportType)
         {
-        case Requests::ReportType::FileLocks:
+        case IStreamerTypes::ReportType::Config:
+            {
+                AZStd::string drivePaths;
+                for (const AZStd::string& path : m_drivePaths)
+                {
+                    drivePaths += path;
+                    drivePaths += ' ';
+                }
```

Comment:
FYI, you can use range based [AZ::StringFunc::Join](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/StringFunc/StringFunc.h#L458-L469) overload to join a range of types convertible to strings to an AZStd::string
```suggestion
                AZ::StringFunc::Join(drivePaths, m_drivePaths, ' ');
```

Reply:
Good reminder. Updated to your suggestion in the next commit.